backend/danswer/background/celery/tasks/tasks.py

```diff
@@ -0,0 +1,48 @@
+from datetime import timedelta
```

Comment:
don't think we should have a tasks/tasks.py?  Also, this file doesn't contain tasks, it has a beat schedule?